tutorials-v4/time-evolution/011_floquet_solver.md

```diff
@@ -0,0 +1,147 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Floquet Solvers
+
+Author: C. Staufenbiel, 2022
+
+### Introduction
+
+The *Floquet formalism* deals with periodic time-dependent systems. It is more efficient to use the Floquet approach to such problems than using the standard master equation solver `qutip.mesolve()`.
+
```

Comment:
just a nitpick, i would say "can be more efficient" and "has a broader range of validity"?